src/extremeweatherbench/utils.py

```diff
@@ -37,6 +37,17 @@
     "surface_temperature": "surface_air_temperature",
 }
 
+POINT_OBS_METADATA_VARS = [
```

Comment:
Should include a comment similar to above ("#: Struct packaging...") explaining what this is.